### DISCUSSION TITLE
Cisco Umbrella

### DISCUSSION THREAD

**Discussion by GGboogie** [3kh0/ext-remover#230] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Does this work for Cisco Umbrella as well? If it doesn't can someone tell me how to unblock Cisco Umbrella?

**Comment by synapt0** [comment]
I believe it is the same as every other page-blocking extension--it only works before version 106.

**Comment by ILikeBypassing** [comment, marked as answer]
use a proxy like holy unblocker
